### PR TITLE
Add meeting notes for 06/09 meeting.

### PR DESCRIPTION
**Pull request by dtig**

### CODE REVIEW

**Comment by dtig** [comment]
@echamudi and @rossberg, Please leave a link, or add slides presented at the meeting to the notes.

**Comment by echamudi** [comment]
Hello @dtig , here's the [Google Slides link](https://docs.google.com/presentation/d/1C_2ZdVep_kt2BZmpLiRJ7VgqFXUG8gUgGUORLghdAGE/edit?usp=sharing) of my presentation.

File: main/2020/CG-06-09.md [hunk 1]
```diff
@@ -40,4 +40,268 @@ Installation is required, see the calendar invite.
...
+
+Clemens Backes
+
+Luke Wagner
+
+Derek Schuff
+
+Alex Crichton
+
+Sven
```
Comment (xtuc) [inline main/2020/CG-06-09.md:112]:
```suggestion
Sven Sauleau
```

**Comment by dtig** [comment]
Leaving an AI for @rossberg here to add slides, merging so notes are available before the next meeting.

**Pull request merged**